pom.xml

```diff
@@ -25,7 +25,7 @@
 
   <properties>
     <vertx.version>4.1.4</vertx.version>
```

Comment:
We also can update dependencies like:
vertx 4.1.4 -> 4.2.4
log4j 2.16.0 -> 2.17.1
mockito 3.12.4 -> 4.2.0
junit 5.7.0 -> 5.8.2
Should we update something within this PR?

Reply:
yes, if it doesn't break anything, please do